DcCore/DcCore/Helper/KeychainManager.swift

```diff
@@ -15,6 +15,11 @@ public class KeychainManager {
     private static let sharedKeychainGroup = "\(KcM.teamId).group.chat.delta.ios"
 
     public static func getAccountSecret(accountID: Int) throws -> String {
+        if let sharedUserDefaults = UserDefaults.shared,
+            !sharedUserDefaults.bool(forKey: "\(UserDefaults.upgradedKeychainEntry)\(accountID)") {
+            upgradeAccountSecret(id: accountID)
```

Comment:
i would skip this upgrade.

- it adds complexity, i like to have less keychain code :)
- most accounts that are theoretically upgradeable are probably broken anyway meanwhile (experimental :)
- the code should be able to work with `kSecAttrAccessibleWhenUnlocked` as well as with `kSecAttrAccessibleAfterFirstUnlocked` anyway (difference is only background fetch from lock-screen when the app is not frozen but terminated)

Reply:
I removed the migration as we discussed before